docs/Configuration.md

```diff
@@ -97,24 +115,34 @@ Available keys : `host`, `user`, `password`, `database`.
 
 ### host
 
+An [LDH domain name] extended with underscore (`_`) and slash (`/`).
```

Comment:
No, that is not a good idea. For zone names it is one thing to extend, but here we have something that should be a hostname that must have either an A or a AAAA record or both. That name should be an LDH name.

Reply:
Okay, I'll add another data type.